src/main/java/com/thecookiezen/bussiness/cluster/entity/DockerHost.java

```diff
@@ -1,15 +1,19 @@
 package com.thecookiezen.bussiness.cluster.entity;
 
 import lombok.Data;
+import lombok.NoArgsConstructor;
 import lombok.NonNull;
 
 import java.io.Serializable;
 import java.time.Instant;
 
 @Data
+@NoArgsConstructor
 public class DockerHost implements Serializable {
     private static final long serialVersionUID = 1L;
 
+    private long id;
```

Comment:
\[Checkstyle\] WARNING: 'member def modifier' have incorrect indentation level 4, expected level should be 2\.